cmd/endpoint.go

```diff
@@ -40,11 +40,12 @@ var EndpointCommand = cli.Command{
 				},
 			},
 			Action: func(c *cli.Context) error {
-				endpointName := c.Args().Get(0)
-				if len(endpointName) == 0 {
-					return cli.NewExitError("endpoint name is required", 1)
+				if len(c.Args()) != 1 {
+					return cli.NewExitError("endpoint is invalid", 1)
```

Comment:
This is quite weird because even if the endpoint name is valid and you add a flag to the end of it, for example:

```
$ bcn endpoint create hello-world --public
endpoint is invalid
```

this would be misleading to the user. The user may infer that dashes are not allowed in the example, even though they are perfectly fine.

Reply:
right let me see if I can fix it 🤔

Reply:
@davidsiaw it seems like that is how this library works currently 😢 
https://github.com/urfave/cli/issues/427

Reply:
@Yuuki77 Ah okay. We can also just make the error message better like, `please place options and flags before the endpoint name.`